kalite/updates/static/js/updates/updates/base.js

```diff
@@ -196,7 +196,7 @@ function updateDisplay(process_name, progress_log) {
         select_update_elements(process_name, ".progressbar-current").progressbar({value: 100*progress_log.stage_percent});
         select_update_elements(process_name, ".progressbar-overall").progressbar({value: 100*progress_log.process_percent});
 
-        select_update_elements(process_name, "#stage-summary").text(sprintf(gettext("Overall progress: %(percent_complete)5.1f%% complete (%(cur_stage)d of %(num_stages)d)"), {
+        select_update_elements(process_name, "#stage-summary").text(sprintf(gettext("Overall progress: %(percent_complete)5.1f%% complete (%(cur_stage)d of %(num_stages)d) "), {
```

Comment:
What I mean is: If you don't change the string as it's read by the translation msg collector, the change might not affect translators. Adding a blank space or newline will definitely change the translation msg. Adding an escape character might not.

Is it because it doesn't work with an escape character?